src/main/java/core/basesyntax/Model/Cat.java

```diff
@@ -1,4 +1,4 @@
-package core.basesyntax.Model;
+package core.basesyntax.model;
```

Comment:
please also change name of the package in lower case (model instead of model ):
src/main/java/core/basesyntax/Model/Cat.java